automation/capitanodoc/index.ts

```diff
@@ -17,37 +17,61 @@
 import * as _ from 'lodash';
 import * as path from 'path';
 
-import capitanodoc = require('../../capitanodoc');
+import { getCapitanoDoc } from './capitanodoc';
 import { Category, Document } from './doc-types';
 import * as markdown from './markdown';
 
-const result: Document = {
-	title: capitanodoc.title,
-	introduction: capitanodoc.introduction,
-	categories: [],
-};
-
-for (const commandCategory of capitanodoc.categories) {
-	const category: Category = {
-		title: commandCategory.title,
-		commands: [],
+/**
+ * Generates the markdown document (as a string) for the CLI documentation
+ * page on the web: https://www.balena.io/docs/reference/cli/
+ */
+export async function renderMarkdown(): Promise<string> {
+	const capitanodoc = await getCapitanoDoc();
+	const result: Document = {
+		title: capitanodoc.title,
+		introduction: capitanodoc.introduction,
+		categories: [],
 	};
 
-	for (const file of commandCategory.files) {
-		// tslint:disable-next-line:no-var-requires
-		const actions: any = require(path.join(process.cwd(), file));
+	for (const commandCategory of capitanodoc.categories) {
+		const category: Category = {
+			title: commandCategory.title,
+			commands: [],
+		};
 
-		if (actions.signature) {
-			category.commands.push(_.omit(actions, 'action'));
-		} else {
-			for (const actionName of Object.keys(actions)) {
-				const actionCommand = actions[actionName];
-				category.commands.push(_.omit(actionCommand, 'action'));
+		for (const file of commandCategory.files) {
+			// tslint:disable-next-line:no-var-requires
+			const actions: any = require(path.join(process.cwd(), file));
```

Comment:
I think there's a smattering of typings for Capitano out there now. Might be worth picking one of them up.

Reply:
I have implemented all other suggestions in this review (new PR to follow) -- thanks! -- but I am not sure I understood this one. I was able to delete the `tslint` annotation but I don't think that's what you meant. Did you mean replacing `typings/capitano.d.ts` with types from elsewhere? By the way, if relevant, note that this file ('index.ts') is also being modified in this other PR pending review: https://github.com/balena-io/balena-cli/pull/1198